spec/views/documents/history/_content_publisher_entry.html.erb_spec.rb

```diff
@@ -25,4 +25,50 @@
                                       text: "Amazing internal note")
     expect(rendered).to have_content(I18n.t!("documents.history.entry_types.internal_note"))
   end
+
+  context "when the timeline entry is for a removal" do
+    it "can show an explanatory_note" do
+      removal = create(:removal, explanatory_note: "My note")
+      timeline_entry = create(:timeline_entry,
+                              entry_type: :removed,
+                              details: removal)
+      render partial: "documents/history/content_publisher_entry",
+             locals: { entry: timeline_entry }
+      expect(rendered).to have_content("My note")
+    end
+
+    it "can show a link to an alternative URL" do
+      removal = create(:removal, alternative_url: "https://example.com")
+      timeline_entry = create(:timeline_entry,
+                              entry_type: :removed,
+                              details: removal)
+      render partial: "documents/history/content_publisher_entry",
+             locals: { entry: timeline_entry }
+      expect(rendered).to have_content("Alternative URL https://example.com",
+                                       normalize_ws: true)
+      expect(rendered).to have_selector("a[href='https://example.com']",
+                                        text: "https://example.com")
+    end
+
+    it "can show a link to an alternative URL that is a path" do
+      removal = create(:removal, alternative_url: "/path")
+      timeline_entry = create(:timeline_entry,
+                              entry_type: :removed,
+                              details: removal)
+      render partial: "documents/history/content_publisher_entry",
+             locals: { entry: timeline_entry }
+      expect(rendered).to have_selector("a[href*='/path']", text: %r{/path\Z})
+    end
+
+    it "can show a redirect" do
+      removal = create(:removal, redirect: true, alternative_url: "https://example.com")
+      timeline_entry = create(:timeline_entry,
+                              entry_type: :removed,
+                              details: removal)
+      render partial: "documents/history/content_publisher_entry",
+             locals: { entry: timeline_entry }
+      expect(rendered).to have_content("Redirected to https://example.com",
+                                       normalize_ws: true)
```

Comment:
It's because the HTML has a new line separating it when output. However since a human would read it as a single line I thought normalize_ws was a nicer option than adding in the new lines.

Reply:
Ah, I see. I noticed that in the view, and was wondering if there's a way to do it inline, which should remove the need for the option here. In the end there didn't seem to be a clear way to change it, apart from something like this...

```
<% prefix = ... %>
<% link = capture ... %>
<%= prefix %> <%= link %>
```

Reply:
Sure, that'd do it. Though I think it ends up being a bit cryptic that it was done to satisfy the tests.

Are you happy for me to merge without further changes?